### PR TITLE
Update to RC2

### PR DESCRIPTION
**Pull request by troyanskiy**
Angular library update to rc2

### CODE REVIEW

File: tools/config/seed.config.ts [hunk 1]
```diff
@@ -299,13 +299,13 @@ export class SeedConfig {
...
     paths: {
       [this.BOOTSTRAP_MODULE]: `${this.APP_BASE}${this.BOOTSTRAP_MODULE}`,
-      '@angular/core': `${this.APP_BASE}node_modules/@angular/core/core.umd.js`,
-      '@angular/common': `${this.APP_BASE}node_modules/@angular/common/common.umd.js`,
-      '@angular/compiler': `${this.APP_BASE}node_modules/@angular/compiler/compiler.umd.js`,
-      '@angular/http': `${this.APP_BASE}node_modules/@angular/http/http.umd.js`,
-      '@angular/router': `${this.APP_BASE}node_modules/@angular/router/router.umd.js`,
-      '@angular/platform-browser': `${this.APP_BASE}node_modules/@angular/platform-browser/platform-browser.umd.js`,
-      '@angular/platform-browser-dynamic': `${this.APP_BASE}node_modules/@angular/platform-browser-dynamic/platform-browser-dynamic.umd.js`,
+      '@angular/core': `${this.APP_BASE}node_modules/@angular/core/index.js`,
```
Comment (mgechev) [inline tools/config/seed.config.ts]: No bundles in rc.2?

Comment (mgechev) [inline tools/config/seed.config.ts]: I'd say that the path should be `./bundles/core.umd.js`.

Comment (troyanskiy) [inline tools/config/seed.config.ts]: Well...
I don'r really understand what does that mean. I just updated my project with rc2 and saw that project can't find ****.umd.js files. So I checked in the @angular/module folder and now there is everywhere index.js file
So I have updated seed.config.ts file and it works now for my project.

File: package.json [hunk 2]
```diff
@@ -115,13 +115,13 @@
...
-    "@angular/platform-browser": "2.0.0-rc.1",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.1",
-    "@angular/router": "2.0.0-rc.1",
+    "@angular/common": "2.0.0-rc.2",
+    "@angular/compiler": "2.0.0-rc.2",
+    "@angular/core": "2.0.0-rc.2",
+    "@angular/http": "2.0.0-rc.2",
+    "@angular/platform-browser": "2.0.0-rc.2",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.2",
+    "@angular/router": "2.0.0-rc.2",
```
Comment (antonybudianto7) [inline package.json]: it should be 3.0.0-alpha.6, 2.0.0 is deprecated

Comment (troyanskiy) [inline package.json]: Wow... Thanks. Checking that.

File: tools/config/seed.config.ts (hunk 1 above)
Comment (mgechev) [inline tools/config/seed.config.ts]: The initial load time is faster. It loads the entire bundle instead of the angular modules one my one.

Comment (troyanskiy) [inline tools/config/seed.config.ts]: Ahh... I see now...
Checking that.
Thank you!

File: package.json [hunk 3]
```diff
@@ -115,13 +115,13 @@
...
-    "@angular/platform-browser": "2.0.0-rc.1",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.1",
-    "@angular/router": "2.0.0-rc.1",
+    "@angular/common": "2.0.0-rc.2",
+    "@angular/compiler": "2.0.0-rc.2",
+    "@angular/core": "2.0.0-rc.2",
+    "@angular/http": "2.0.0-rc.2",
+    "@angular/platform-browser": "2.0.0-rc.2",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.2",
+    "@angular/router": "3.0.0-alpha.3",
```
Comment (TheDonDope) [inline package.json]: Needs to be 3.0.0-alpha.**6**

Comment (troyanskiy) [inline package.json]: Oops

**Pull request closed without merging**